<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::ProjectorWidget</class>
 <widget class="QWidget" name="omni::ui::ProjectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>252</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Projector</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="omni::ui::ProjectorGroupBox" name="groupBox">
     <property name="title">
      <string>Projector #1</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QFrame" name="settingsFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="yawAngleLabel">
           <property name="toolTip">
            <string>Yaw Angle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/PROJ_YAW.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="boxYawAngle">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="distanceCenterLabel">
           <property name="toolTip">
            <string>Distance Center</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/DIST_CENT_SMALL.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="towerHeightLabel">
           <property name="toolTip">
            <string>Tower Height</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/TOWER_HEIGHT_SMALL.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="boxTowerHeight">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-12.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="shiftLabel">
           <property name="toolTip">
            <string>Tower shift</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/PROJ_SHIFT.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="boxShift">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-2.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="pitchAngleLabel">
           <property name="font">
            <font/>
           </property>
           <property name="toolTip">
            <string>Pitch Angle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/PROJ_PITCH.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="boxPitchAngle">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="rollAngleLabel">
           <property name="toolTip">
            <string>Roll Angle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/PROJ_ROLL.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="boxRollAngle">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="deltaYawAngleLabel">
           <property name="toolTip">
            <string>Delta Yaw Angle</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/buttons/rc/buttons/YAw_small.svg</pixmap>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="boxDeltaYawAngle">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="minimum">
            <double>-45.000000000000000</double>
           </property>
           <property name="maximum">
            <double>45.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="boxDistanceCenter">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="maximum">
            <double>30.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omni::ui::ProjectorGroupBox</class>
   <extends>QGroupBox</extends>
   <header>omni/ui/ProjectorGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/style.qrc"/>
  <include location="style.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>boxDeltaYawAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxDistanceCenter</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxTowerHeight</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxShift</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxPitchAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxRollAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxYawAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>updateSettings(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>toggleSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>projectorVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxYawAngle</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>yaw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxDistanceCenter</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>distance_center()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxTowerHeight</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>tower_height()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxShift</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>shift()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxPitchAngle</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>pitch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxRollAngle</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>roll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxDeltaYawAngle</sender>
   <signal>editingFinished()</signal>
   <receiver>omni::ui::ProjectorWidget</receiver>
   <slot>delta_yaw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleSettings()</slot>
  <slot>updateSettings(double)</slot>
  <slot>projectorVisible(bool)</slot>
  <slot>yaw()</slot>
  <slot>distance_center()</slot>
  <slot>tower_height()</slot>
  <slot>shift()</slot>
  <slot>pitch()</slot>
  <slot>roll()</slot>
  <slot>delta_yaw()</slot>
 </slots>
</ui>
